src/main/java/jeresources/profiling/ProfilingTimer.java

```diff
@@ -81,6 +81,6 @@ public long getBlocksPerLayer(int dim) {
     }
 
     private static String getDimensionName(int dim) {
-        return TranslationHelper.tryDimensionTranslate(DimensionManager.getProvider(dim).getDimensionType().getName());
+        return TranslationHelper.tryDimensionTranslate(DimensionManager.getWorld(dim).provider.getDimensionType().getName()) + " (" + dim + ")";
```

Comment:
`DimensionManager.getProvider(dim)` and `DimensionManager.getWorld(dim).provider` are the exact same code. Why make it more complex.

Reply:
There was a subtle difference in the lookup that caused the wrong output.  I'll make sure that is really true though.